lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -83,6 +83,12 @@ def self.available_options
                                        optional: true,
                                        default_value: false,
                                        is_string: false),
+          FastlaneCore::ConfigItem.new(key: :is_draft,
+                                       env_name: 'GHHELPER_CREATE_RELEASE_IS_DRAFT',
+                                       description: 'True if this is a draft release',
```

Comment:
💄  Nit:
```suggestion
                                       description: 'True to create the GitHub release as a draft (instead of publishing it immediately)',
```